internal/github/client.go

```diff
@@ -17,8 +18,11 @@ const (
 	pullRequestFilter = "closed"
 )
 
+type ctxKey struct{}
+
 var (
-	ErrMissingOriginRemote = errors.New("remote 'origin' is not configure")
+	GitHubHttpClientCtxKey = ctxKey{}
```

Comment:
Similar trick here to control the http client that `go-github` uses. We need to specify the `go-vcr` client in order to capture and replay responses. 

I don't like to change production code for tests, but this seems like a better compromise than the alternatives that I could think of.